edtrio-client/src/graphqlOperations/generated-types/createMultipleChoiceAnswer.ts

```diff
@@ -0,0 +1,19 @@
+/* tslint:disable */
+// This file was automatically generated and should not be edited.
+
+// ====================================================
+// GraphQL mutation operation: createMultipleChoiceAnswer
+// ====================================================
+
+export interface createMultipleChoiceAnswer_createMultipleChoiceAnswer {
```

Comment:
just for my understanding: How do we end up with these super long names?

Reply:
It's done by some Prisma magic, maybe @Atyanar knows more?

Reply:
Oh, yeah, of course. That's the command we ran to download the types in the client. I remember...